archivist_samples/wipp/README.md

```diff
@@ -95,16 +104,16 @@ to the Drum and/or Cask Assset using `Wipp.read()`:
 
 ```python
 # Assume Archivist connection already initialized in `arch`
-Drum = Wipp(arch)
+Drum = Wipp(arch, "55 gallon drum")
 drum.read("assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx")
 ```
 
 If you do not know the RKVST Asset Identity then you can load data based on any unique set of attributes using `Wipp.read_by_signature()`:
 
 ```python
 # Assume Archivist connection already initialized in `arch`
-Drum = Wipp(arch)
-drum.read_by_signature({"wipp_package_id": "gklh588i8wd9"})
+Drum = Wipp(arch, "55 gallon drum")
+drum.read_by_signature({"wipp_package_id": "wipp"})
```

Comment:
This change (removing the random suffix) essentially means you can only ever do this once. Is that the intention?

Reply:
Hi @JAG-UK   . . . to match the code changes yep. . . one can only run the sample once based on test_namespace.  Per our conversation basically a unique id  . . . if someone does not update their namespace then it won't run.  The default namespace is "wipp" thus if someone runs with no namespace it will only run once.

Will update the overall readme to ensure that the test_namespace is a unique value and one has to update this value in order to run multiple instances.